src/app/core/entity-components/historical-data/historical-data/historical-data.component.spec.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  ComponentFixture,
+  fakeAsync,
+  TestBed,
+  tick,
+} from "@angular/core/testing";
+
+import { HistoricalDataComponent } from "./historical-data.component";
+import { NoopAnimationsModule } from "@angular/platform-browser/animations";
+import { EntityMapperService } from "../../../entity/entity-mapper.service";
+import { HistoricalDataModule } from "../historical-data.module";
+import { ColumnDescriptionInputType } from "../../entity-subrecord/column-description-input-type.enum";
+import { Entity } from "../../../entity/entity";
+import { HistoricalEntityData } from "../historical-entity-data";
+import moment from "moment";
+import { DatePipe } from "@angular/common";
+
+describe("HistoricalDataComponent", () => {
+  let component: HistoricalDataComponent;
+  let fixture: ComponentFixture<HistoricalDataComponent>;
+  let mockEntityMapper: jasmine.SpyObj<EntityMapperService>;
+
+  beforeEach(async () => {
+    mockEntityMapper = jasmine.createSpyObj(["loadType"]);
+    mockEntityMapper.loadType.and.resolveTo([]);
+
+    await TestBed.configureTestingModule({
+      declarations: [HistoricalDataComponent],
+      imports: [HistoricalDataModule, NoopAnimationsModule],
+      providers: [
+        { provide: EntityMapperService, useValue: mockEntityMapper },
+        DatePipe,
+      ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(HistoricalDataComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it("should create", () => {
+    expect(component).toBeTruthy();
+  });
+
+  it("should add mapping function to configurable enum types", () => {
```

Comment:
shouldn't this be functionality of the entity-subrecord.component rather than the historical-data?

and maybe clearer
```suggestion
  it("should add mapping function to display label for configurable enum types", () => {
```